pom.xml

```diff
@@ -53,6 +53,7 @@
         <org.apache.avalon.framework.version>4.3.1</org.apache.avalon.framework.version>
         <hamcrest.version>2.2</hamcrest.version>
         <hibernate.version>5.6.15.Final</hibernate.version>
+        <hibernate-search.version>6.2.4.Final</hibernate-search.version>
```

Comment:
Already old version and support for this version is already limited.

Reply:
That's true, but our Hibernate version is also old. I tried a newer version first, but it didn't work. Either the Hibernate version needs to be updated first, or both together.

Reply:
This is true. Switching to a new Hibernate Version is not easy as they already go the way to Java 17 and newer including JavaX to JakartaEE namespace change. There are version available which could be used as they interact as a version between both worlds and can be used while switching the application to Java 17 and newer but this is even not a trivial task.

With the integration of Hibernate-Search it would not be easier as 2 frameworks must be changed.